src/main/jbake/content/docs/components/btm/clientconfig.adoc

```diff
@@ -0,0 +1,81 @@
+= Client Configuration Guide
+Gary Brown
+2015-06-30
+:description: Hawkular Business Transaction Management
+:icons: font
+:jbake-type: page
+:jbake-status: published
+
```

Comment:
You may perhaps want to add a toc

add this to the header and start of content:

```
:jbake-status: published
:icons: font
:toc: macro
:toc-title:

toc::[]
```